### PR TITLE
Run swiftlint in ci

### PR DESCRIPTION
**Pull request by pixyzehn**
[WIP] Fixed https://github.com/JohnSundell/Marathon/issues/84

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,15 @@
-os: linux
+os:
+    - linux
+    - osx
```
Comment (JohnSundell) [inline .travis.yml]: We don't want to run Travis for macOS, we have BuddyBuild for that 🙂

Comment (pixyzehn) [inline .travis.yml]: Sure, thank you! I'll do again:)

**Comment by JohnSundell** [comment]
@pixyzehn You can just remove the `.travis.yml` changes from the PR and we can merge it in - the BuddyBuild changes looked great 👍

**Comment by pixyzehn** [comment]
@JohnSundell Yes, I think so too. I'll do so later! Thank you:)

**Comment by pixyzehn** [comment]
Done:) https://github.com/JohnSundell/Marathon/pull/89

**Pull request closed without merging**